docs/en/02_Developer_Guides/14_Files/02_Images.md

```diff
@@ -200,13 +200,112 @@ SilverStripe\Core\Injector\Injector:
+on _web.dev_ for more information.
+
+#### Selectively disabling lazy loading
+
+Images that are expected to be initially visible on page load, should be _eager_ loaded. This 
+provides a small performance gain since the browser doesn't have to render the entire page layout 
+before determining if the images need to be loaded. When in doubt, it's usually preferable to lazy 
+load the image.
+
+Images that don't have dimensions should not be lazy loaded as that might alter the layout of the 
+page after the initial page load. 
+
+Content authors have the ability to selectively disable lazy loading when inserting images in an 
+HTML editor field.
+
+Developers can selectively disable lazy loading for individual image in a SS template by calling 
+`LazyLoad(false)` on the image variable. Developers can allow content authors to control the 
+loading attribute of a specific image by adding lazy load field next to the 
+[`UploadField`](api:SilverStripe\Assets\UploadField).
+
+```php
+<?php
+use SilverStripe\AssetAdmin\Forms\UploadField;
+use SilverStripe\Assets\Image;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Forms\DropdownField;
+
+class Page extends SiteTree
+{
+    private static $db = [
+        'LogoLoading' => 'Boolean'
+    ];
+
+    private static $has_one = [
+        'Logo' => Image::class
+    ];
+
+    private static $defaults = [
+        'LogoLoading' => true
+    ];
+
+
+    public function getCMSFields()
+    {
+        $fields = parent::getCMSFields();
+
+        $loadingSource = [
+            true => 'Lazy (Default)',
+            false => 'Eager'
+        ];
+        
+        $fields->addFieldsToTab(
+            'Root.Main',
+            [
+                UploadField::create('Logo'),
+                DropdownField::create('LogoLoading', 'Loading', $loadingSource)
+            ]
+        );
+
+        return $fields;
+    }
+}
+```
+
+#### Controlling lazy loading in SS templates
+
+If you are manually writting `<img>` tags in your SS template, those images will not be automatically
+lazy loaded. You will need to add the `loading="lazy"` attribute yourself if you want the image to be
+lazy loaded.
+
+```html
+<h1>Recommended way of rendering Images</h1>
+<!-- Image will be lazy loaded -->
+$Logo
+
+<!-- Image will NOT be lazy loaded -->
+$Logo.LazyLoad(false)
+
+<!-- We're allowing content authors to choose if the image is eager loaded-->
+$Logo.LazyLoad($LogoLoading)
+
+<h1>Manual way of rendering Images</h1>
+
+<img src="$Logo.URL" width="$Logo.Width" width="$Logo.Height" loading="lazy" alt="Company Logo" />
+
+<!-- The size of this image is controlled by a CSS class so it can be lazy loaded -->
+<img src="$resourceURL('themes/example/images/footer.png')" class="64x64square" loading="lazy" alt="" />
+
+<!-- We don't have dimension for this image, so we eager load it -->
+<img src="//example.com/sponsor.webp" alt="A generous sponsor" />
+```
+
+#### Disabling lazy loading globally
```

Comment:
I see completly disabling lazy loading as an edge case that very few devs will consider doing past the 4.9 release. Since this is the permanent doc, I think it makes sense to treat it as a secondary concern.

Might make sense to move it up in the changelog however.